requirements.txt

```diff
@@ -1,16 +1,34 @@
+alembic==1.5.7
+cachelib==0.1.1
+certifi==2020.12.5
+chardet==4.0.0
 click==6.6
 Flask==1.1.2
+Flask-Migrate==2.7.0
 Flask-OAuthlib==0.9.6
 Flask-SQLAlchemy==2.4.4
+greenlet==1.0.0
 gunicorn==19.6.0
-httplib2==0.9.2
-itsdangerous==0.24
+httplib2==0.19.0
+idna==2.10
+importlib-metadata==3.7.3
+install==1.3.4
+itsdangerous==1.1.0
 Jinja2==2.11.3
-MarkupSafe==0.23
+Mako==1.1.4
+MarkupSafe==1.1.1
 oauthlib==2.1.0
+pip==18.1
 psycopg2==2.7.4
+pyparsing==2.4.7
+python-dateutil==2.8.1
+python-editor==1.0.4
 requests==2.25.1
 requests-oauthlib==1.1.0
-SQLAlchemy==1.0.14
+setuptools==40.6.2
+six==1.15.0
+SQLAlchemy==1.4.0
```

Comment:
It turns out that SQLAlchemy introduced a breaking change in 1.4: they will only accept PostgreSQL database URLs that start with `postgresql://`, not `postgres://`; see discussion in issue https://github.com/sqlalchemy/sqlalchemy/issues/6083 .

Since our `DATABASE_URL` comes from Heroku, we either need to [preprocess it](https://help.heroku.com/ZKNTJQSK/why-is-sqlalchemy-1-4-x-not-connecting-to-heroku-postgres) before handing it off, or pin to an older version.

Do we need the latest version of SQLAlchemy? Can we use the latest 1.3 version instead?

If not, we'll need to implement the URL translation described in the Heroku help article, and update the README.

Reply:
I don't think we need the latest version of SQLAlchemy specifically. We can also change the config var for `DATABASE_URL` in the Heroku dashboard. Would you rather do that?

Reply:
> We can also change the config var for `DATABASE_URL` in the Heroku dashboard. Would you rather do that?

The value in `DATABASE_URL` is [populated automatically](https://devcenter.heroku.com/articles/heroku-postgresql#provisioning-heroku-postgres) by the [heroku-postgresql add-on](https://elements.heroku.com/addons/heroku-postgresql), and can change over time due to upgrades, scheduled maintenance, failovers, and the like. If you see a way to change how that works, please say more! But if you're suggesting manually editing the config variable, I would rather not do that, because it will lead to a surprising outage sometime much later.

I think I would rather downgrade to SQLAlchemy 1.3.x, at least for now. We can probably wait until Heroku offers a migration path, or we can manually implement the workaround, but either way let's do that outside of this PR.

Reply:
I was deceived by the easy availability of the config variables - I didn't realize they were populated automatically. We can stick with SQLAlchemy 1.3.0 in that case. I'll add a commit with an updated `requirements.txt`.